django/api/management/commands/populate_db.py

```diff
@@ -132,7 +133,7 @@ def register_riot_apps():
                     'Authorization': 'Token ' + token,
                 }
                 payload = {
-                    'name': application_name,
+                    'name': slugify(application_name),
```

Comment:
Why do we want to slugify the application's name?

Reply:
oops, it's a leftover. Thanks for the catch